cadt/defaults/main.yml

```diff
@@ -22,10 +36,13 @@ cadt_governance_body_id: 23f6498e015ebcd7190c97df30c032de8deb5c8934fc1caa928bc31
 chia_data_layer_rpc_port: 8562
 chia_wallet_rpc_port: 9256
 
+# Datalayer HTTP location, including schema (http:// or https://).
+datalayer_http_location: "http://{{ cadt_api_hosts | first }}:8575"
+
 # Chia network
-network: "mainnet"
+cadt_network: "mainnet"
```

Comment:
In the chia-blockchain role, this var is just `network` - Usually, I keep the var the same so that I can set network once and have it apply to both roles. Is there a case where network for cadt wont match the network set for the chia-blockchain role?

Reply:
seems like for testnet its just `testnet` instead of more specific `testnet10`. if thats the case, then comment can just be resolved.

Reply:
yep - that's the case and why I changed it.  Which drives me crazy.  I learned that the CADT var for `network` is actually a legacy thing and can likely be removed in a future release, so not making a fuss to change it to be easier for the automation.